src/ui/Styles/patterns/_success-summary.scss

```diff
@@ -0,0 +1,22 @@
+.error-summary-heading:only-child {
```

Comment:
Could you comment where this code came from? Was it hand cranked or pulled from somewhere? Putting it in the commit comment would suffice.

Reply:
I'm going to clean this up quickly